test/hotspot/jtreg/runtime/CommandLine/PrintTouchedMethods.java

```diff
@@ -23,7 +23,8 @@
 
 /*
  * @test
- * @bug 8025692
+ * @requires vm.flavor != "zero"
+ * @bug 8025692 8273333
```

Comment:
Nit: these lines tend to be in the other order

Reply:
All right, changed.